charts/bitbucket-runner-autoscaler/values.yaml

```diff
@@ -0,0 +1,63 @@
+namespace: "bitbucket-runner-controller"
+
+cleaner:
+  enabled: true
+  image: bitbucketpipelines/runners-autoscaler
+  tag: "3.7.0"
+
+controller:
+  enabled: true
+  image: bitbucketpipelines/runners-autoscaler
+  tag: "3.7.0"
+
+tolerations: []
```

Comment:
I would add a global section that would apply this to all applicable resources and make it overridable per instance (controller, cleanup controller, runners)